src/3d/qgstiledscenechunkloader_p.cpp

```diff
@@ -168,32 +169,24 @@ QgsChunkLoader *QgsTiledSceneChunkLoaderFactory::createChunkLoader( QgsChunkNode
   return new QgsTiledSceneChunkLoader( node, mIndex, *this, mZValueScale, mZValueOffset );
 }
 
-// converts box from map coordinates to world coords (also flips [X,Y] to [X,-Z])
-static QgsAABB aabbConvert( const QgsBox3D &b0, const QgsVector3D &sceneOriginTargetCrs )
-{
-  const QgsBox3D b = b0 - sceneOriginTargetCrs;
-  return QgsAABB( b.xMinimum(), b.zMinimum(), -b.yMaximum(), b.xMaximum(), b.zMaximum(), -b.yMinimum() );
-}
-
 QgsChunkNode *QgsTiledSceneChunkLoaderFactory::nodeForTile( const QgsTiledSceneTile &t, const QgsChunkNodeId &nodeId, QgsChunkNode *parent ) const
 {
   QgsChunkNode *node = nullptr;
   if ( hasLargeBounds( t, mBoundsTransform ) )
   {
     // use the full extent of the scene
-    QgsVector3D v0 = mRenderContext.mapToWorldCoordinates( QgsVector3D( mRenderContext.extent().xMinimum(), mRenderContext.extent().yMinimum(), -100 ) );
-    QgsVector3D v1 = mRenderContext.mapToWorldCoordinates( QgsVector3D( mRenderContext.extent().xMaximum(), mRenderContext.extent().yMaximum(), +100 ) );
-    QgsAABB aabb( v0.x(), v0.y(), v0.z(), v1.x(), v1.y(), v1.z() );
+    QgsVector3D v0( mRenderContext.extent().xMinimum(), mRenderContext.extent().yMinimum(), -100 );
```

Comment:
Was dropping the mapToWorldCoordinates conversion intentional here?

Reply:
It was intentional - we want QgsBox3D bounding boxes of chunk nodes to be always in map coordinates (rather than in "3D world" coordinates as used in QgsAABB)